src/pinning.js

```diff
@@ -150,6 +152,20 @@ class Pinning {
     this.analytics.trackOpenDB(address, timer.timers.openDB.duration())
   }
 
+  invalidateDBCache (odbAddress, rootStoreAddress) {
+    const split = odbAddress.split('.')
```

Comment:
a regexp would be more explicit

Reply:
Idk, I tend to think that regexps are harder to read and understand for beginners 🙂